### DISCUSSION TITLE
Can I use NvChad telescope color theme ?

### DISCUSSION THREAD

**Discussion by BearyDevs** [LazyVim/LazyVim#2762] [Q&A]
Can I use NvChad telescope color theme ?

And how to config it. Thank you

**Comment by dpetka2001** [comment]
If NvChad provides its colorscheme as an independent plugin, then you can definitely install it. If the colorscheme is bound to NvChad itself, then there's not much to be done.

**Comment by est7** [comment]
https://github.com/kingavatar/nvchad-ui.nvim
